Neos.Flow/Tests/Unit/Session/SessionTest.php

```diff
@@ -300,8 +301,9 @@ public function startPutsACookieIntoTheHttpResponse()
      */
     public function startThrowsAnExceptionIfIncompatibleRequestHandlerIsUsed()
     {
+        $mockRequestHandler = $this->createMock(RequestHandlerInterface::class);
         $mockBootstrap = $this->createMock(Bootstrap::class);
-        $mockBootstrap->expects($this->any())->method('getActiveRequestHandler')->will($this->returnValue(new \stdClass()));
+        $mockBootstrap->expects($this->any())->method('getActiveRequestHandler')->willReturn($mockRequestHandler);
```

Comment:
Is this related?

Reply:
It needed to return an instanceof RequestHandlerInterface due to the typehints and then I also just switched to the more readable `->willReturn` in favor over `->will($this->returnValue(`